.gitignore

```diff
@@ -1,3 +1,2 @@
 /node_modules
-/.nuxt
```

Comment:
Why remove this?

Reply:
Because the nuxt site that is used in the tests is now built and generated in a separate directory in `test/tmp/`, instead of the root directory.
See https://github.com/njam/nuxt-i18n-module/pull/12/files#diff-2460143cb26f0ed97dcd290a828d3a98